src/pivotal-ui/components/svg-iconography/comment-box.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 24 21" xmlns="http://www.w3.org/2000/svg">
+  <title>
```

Comment:
Likely no. I did some copy-pasta from the asset provided and some hyper quick removal of attrs that were blocking me (not looking at others). I can go through and be more thorough on that.